README.md

```diff
@@ -5,6 +5,22 @@ The LightStep distributed tracing library for C++.
 
 ## Installation
 
+The library supports being built in several configurations to support a variety of uses.  The default build configuration includes gRPC support for sending data to LightStep, or alternately the library may be configured without gRPC to use a user-defined transport library. The library can be built with support dynamic loading, for applications that support more than one OpenTracing-compatible tracer.
+
+### Requirements
+
+To build and install the LightStep distributed tracing library, you will need to have several tools and libraries intalled.
+
+1. cmake
+1. protobuf
+1. grpc
```

Comment:
The numbers are all 1.

Reply:
This is markdown, the document processor will fix it.